### PR TITLE
Newsletter signup in wizard

### PR DESCRIPTION
**Pull request by mikejolley**
Adds a newsletter form to the wizard:

2017-08-08 at 18 00

@MarinaWoo Could you review the copy? Is this everything you had in mind?

### CODE REVIEW

File: includes/admin/class-wc-admin-setup-wizard.php [hunk 1]
```diff
@@ -929,6 +929,13 @@ public function wc_setup_ready() {
 			</div>
 		<?php endif; ?>
 
+		<div class="woocommerce-message woocommerce-newsletter">
+			<p><?php esc_html_e( 'Join our mailing list to get regular updates on WooCommerce, tips for managing your store and inspiration from the wide world of Woo!', 'woocommerce' ); ?></p>
+			<form action="//woocommerce.us8.list-manage.com/subscribe/post?u=2c1434dc56f9506bf3c3ecd21&amp;id=13860df971" method="post" target="_blank" novalidate>
+				<input type="email" value="<?php echo esc_attr( $current_user->user_email ); ?>" name="EMAIL" placeholder="<?php esc_attr_e( 'Email address', 'woocommerce' ); ?>" required><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button-primary button button-large">
```
Comment (claudiosanches) [inline includes/admin/class-wc-admin-setup-wizard.php]: Allow translate the placeholder in the input, but not the value on the button:

```
value="Subscribe"
```

**Review by claudiosanches — CHANGES_REQUESTED** [review]
Looks good, just missing a hard coded string.

**Comment by claudiulodro** [comment]
screen shot 2017-08-08 at 2 27 14 pm

Not really a part of your PR (which looks good to go), but the subject line and contact email should probably be updated. Who can we hassle about that? :)

**Comment by MarinaWoo** [comment]
Seeing it in situ, can you change the sign up copy to:

_Join the WooCommerce mailing list for help getting started, tips and product updates_

@claudiulodro I will sort out the MailChimp subject line and contact email, TY!

**Pull request merged**